kernel/fde/fde_test.go

```diff
@@ -528,6 +528,62 @@ func (s *fdeSuite) TestRevealErr(c *C) {
 	c.Check(osutil.FileExists(filepath.Join(dirs.GlobalRootDir, "/run/fde-reveal-key")), Equals, false)
 }
 
+func (s *fdeSuite) TestDeviceUnlock(c *C) {
+	checkSystemdRunOrSkip(c)
```

Comment:
actually same, do we need this check either since we are mocking systemd-run now?

Reply:
I don't recall the state of the unit tests when I originally reviewed this but we today have misleading variable names where we do 

```go
	mockSystemdRun := testutil.MockCommand(c, "fde-device-unlock", fmt.Sprintf(`
```

but note that we are not mocking systemd-run, we are mocking the hook fde-device-unlock. So it makes sense that we still do this here.